internal/cli/wire.go

```diff
@@ -113,12 +114,13 @@ func wireThreads(ctx context.Context) (Threads, error) {
 }
 
 type Threads struct {
-	hud   hud.HeadsUpDisplay
-	upper engine.Upper
+	hud       hud.HeadsUpDisplay
+	upper     engine.Upper
+	tiltBuild model.TiltBuild
```

Comment:
i... _assume_ this is an appropriate way to use threads? i.e. for stuff you want to be able to access after all the wiring is complete?